core/web/config_controller.go

```diff
@@ -25,6 +25,16 @@ func (cc *ConfigController) Show(c *gin.Context) {
 	jsonAPIResponse(c, cw, "config")
 }
 
+func (cc *ConfigController) Dump(c *gin.Context) {
+	tomlStr, err := cc.App.ConfigDump(c)
+	if err != nil {
+		cc.App.GetLogger().Errorw("Failed to dump TOML config", "err", err)
+		jsonAPIError(c, http.StatusInternalServerError, err)
+		return
+	}
+	jsonAPIResponse(c, tomlStr, "config") //TODO different resource name?
```

Comment:
I don't fully grok the significance of these. If it just determines a field name in the json, then nbd to reuse :shrug: 
@vyzaldysanchez